tests/Unit/Core/Form/IdentifiableObject/DataFormatter/BulkCombinationFormDataFormatterTest.php

```diff
@@ -136,7 +136,32 @@ public function getDataToFormat(): iterable
             ],
         ];
 
-        yield 'partial stock data wit no options' => [
+        yield 'stock dat with fixed quantity' => [
```

Comment:
```suggestion
        yield 'stock data with fixed quantity' => [
```
Just small typo suggestion